### ISSUE TITLE
error: only one tx sender is allowed within one bundle

### ISSUE THREAD

**Issue by Ruud88x** [node-real/js-direct-route#6]
In my use case I'm trying to send two tx in one bundle.
Tx1 is a pendingTx I'm looking to resubmit. This is a Tx of another wallet.
Tx2 is my own Tx I wish to bundle with Tx1.

When sending this bundle, nodereal returns this error:

`error: only one tx sender is allowed within one bundle`

Why is this limitation in place? Is this working as intended? If you can only use it to send Tx from one sender, what is the use of the service? You might as well use a smart contract to "bundle" your onchain logic in this case.

**Comment by a04512** [comment]
because someone uses this for sandwich attack

**Comment by Ruud88x** [comment]
> because someone uses this for sandwich attack

Cool, but the readme details a usecase where you bundle two different senders as well.

**Comment by yoongyy** [comment]
I get another error:
Returned error: simulate tx success, while status of receipt is failed.

When I try to get validators status, the result is below:
{
  status: 0,
  validators: { MathWallet: 0, NodeReal: 0, 'TW Staking': 0 }
}

Is that means no validator is active now in NodeReal?

**Comment by eniolasonowo** [comment]
I have this issue also,  I want to perform a wallet rescue, but I need to send more than one tx from multiple wallets in a single bundle.

**Comment by greyireland** [comment]
It didn't figure out what it wanted to do,It should die soon

**Comment by LubomirKurpel** [comment]
> I get another error: Returned error: simulate tx success, while status of receipt is failed.
> 
> When I try to get validators status, the result is below: { status: 0, validators: { MathWallet: 0, NodeReal: 0, 'TW Staking': 0 } }
> 
> Is that means no validator is active now in NodeReal?

No, according to docs, 0 means active and 1 means inactive. Which is kind of weird but thats it. So basicly only 3 validators support this nowdays.

https://docs.nodereal.io/nodereal/direct-route/user-guide/for-traders/direct-route-api#eth_validatorstatus

**Comment by moinhosec** [comment]
I have this issue also
